packages/vuetify/src/services/theme/index.ts

```diff
@@ -202,6 +213,24 @@ export class Theme extends Service {
     }
   }
 
+  private applyVueMeta23 () {
+    const { set, remove } = this.vueMeta.addApp('vuetify')
+
+    // remove the old style sheet first
+    if (this.hasMeta) {
+      remove()
+    }
+    set({
+      style: [{
+        cssText: this.generatedStyles,
+        type: 'text/css',
+        id: 'vuetify-theme-stylesheet',
+        nonce: (this.options || {}).cspNonce,
+      }],
+    })
+    this.hasMeta = true
```

Comment:
@pimlie does this seem like an OK way to make things reactive? `applyVueMeta23()` get triggered by a watcher

Reply:
Calling `remove` is actually not required here, set always overwrites all the previous meta info. See the [implementation](https://github.com/nuxt/vue-meta/blob/master/src/shared/additional-app.js) in vue-meta.

`remove` is only required to call if you want to remove all meta data without setting new.

Reply:
sweet, that makes it a lot easier